language/solana/examples/basic/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "basic"
+version = "1.0.0"
+
+[addresses]
+std = "0x1"
```

Comment:
This is not a good number for testing hex, better to have something like `0xDEAD`.

Reply:
It seems std library by convention is 0x1, but I added an address for basic package.